src/pipecat/services/aws/tts.py

```diff
@@ -253,7 +253,14 @@ def read_audio_data(**args):
 
             yield TTSStartedFrame()
 
-            CHUNK_SIZE = 1024
+            # We download a few seconds of audio before we start pushing the
+            # first audio frame. This will make sure we download the rest of the
+            # audio while audio is being played without causing audio glitches
+            # (specially at the beginning). Of course, this will also depend on
+            # how fast the TTS service generates bytes.
+            CHUNK_SECONDS = 1
+            CHUNK_SIZE = self.sample_rate * CHUNK_SECONDS * 2  # 2 bytes/sample
```

Comment:
nit: Inside the `BaseOutputTransport`, we created a property called `audio_chunk_size`. Wouldn't it be worth doing the same here for the `TTSService`, just to avoid duplicating this code across all tts services ?

Reply:
Yes, I'll do that!

Reply:
Updated. Please, take another look.